packages/react/tsconfig.json

```diff
@@ -1,4 +1,4 @@
 {
   "extends": "../../tsconfig.base.json",
-  "include": ["./src"]
+  "include": ["./src", "../react-sandbox/src/_lib/ComponentAbstraction.tsx"]
```

Comment:
Is this change something that was left over when the file was moved?
I didn't think I needed it!

Reply:
Thank you. That was a mistake.